static/templates/settings/alert_word_settings_item.hbs

```diff
@@ -1,13 +1,15 @@
 {{! Alert word in the settings page that can be removed }}
-<li class="alert-word-item" data-word='{{word}}'>
-    <div class="alert-word-information-box grey-box">
+
+<tr id="alert-word-item" class="alert-word-item" data-word='{{word}}'>
```

Comment:
It’s not valid to have multiple elements with the same `id`.

Reply:
Okay thanks so much! Will make these changes and repush. I've got the updating pull requests down so more issues with that. Will make these changes!